components/artist_page_cta/view.coffee

```diff
@@ -56,6 +57,10 @@ module.exports = class ArtistPageCTAView extends Backbone.View
     analyticsHooks.trigger 'artist_page:cta:shown'
     setTimeout (=> @disableScroll()), 400
 
+  keyAction: (e) =>
+    if e.keyCode == 27 || e.which == 27
```

Comment:
If you're using `e.which` I don't think you need `e.keyCode` part at all. #minor